packages/rs-platform-version/src/version/fee/data_contract/mod.rs

```diff
@@ -10,3 +13,40 @@ pub struct FeeDataContractValidationVersion {
     pub document_type_base_unique_index_fee: u64,
     pub document_type_unique_index_per_property_fee: u64,
 }
+
+impl FeeDataContractValidationVersion {
+    pub(crate) fn to_hash(&self) -> u64 {
+        let mut hasher = Sha256::new();
+        Digest::update(&mut hasher, &self.document_type_base_fee.to_be_bytes());
```

Comment:
It's not solving the problem if you add a field to FeeDataContractValidationVersion and forget to update this function. Hashing it's a bit slow to use for comparison. I would stay with a comparison of all properties. I wrote the solution for this problem in the other comment above.